app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -12,6 +13,14 @@ export const ListingDetailsAdditionalInformation = ({
   listing,
   imageSrc,
 }: ListingDetailsAdditionalInformationProps) => {
+  const getCommissionString = () => {
+    return listing.Realtor_Commission_Unit === "percent"
+      ? t("listings.realtorCommissionPercentage", {
```

Comment:
let's fix the spacing for this text prop. looks like tl has no space, zh has two spaces and the others have one space